flex-backend/websockets.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
This is just to test out the websocket. It has a couple buttons to request the list of movies, and one that you can use to send a specific movie to test with